app/auth/validate.py

```diff
@@ -0,0 +1,45 @@
+from flask_jwt import JWT
+from functools import wraps
+from werkzeug.exceptions import BadRequest
+from flask import request, jsonify, Blueprint, abort
+
+jwt = JWT()
+
+
+# Validate all requests
+def validate_json(*expected_args):
+    def wrapper(f):
+        @wraps(f)
+        def decorated_function(*args, **kwargs):
+            if not request.is_json:
+                msg = "Request must be a valid json"
+                return jsonify({"error": msg}), 415
+            try:
+                data = request.get_json()
+                for argument in expected_args:
+                    if (argument not in data or not data.get(argument) or
```

Comment:
These seem to do exactly the same thing, you can just use 1 of them: 
`argument not in data` and  `not data.get(argument)`